common.blocks/i-bem/__collection/i-bem__collection.js

```diff
@@ -179,7 +179,7 @@ var BemCollection = inherit(/** @lends BemCollection.prototype */{
                 arg instanceof BemCollection?  arg._entities : arg);
         }
 
-        return new BemCollection(arrayConcat.apply(this._entities, argsForConcat));
+        return new BemCollection(Array.prototype.concat.apply(this._entities, argsForConcat));
```

Comment:
This variable used just once, but placed in separated var statement that causes jscs error.